app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -371,18 +378,28 @@ fun performKeyAction(
             // Check here for the action #s:
             // https://developer.android.com/reference/android/R.id
 
-            // Select all
-            ime.currentInputConnection.performContextMenuAction(16908319)
+            ime.currentInputConnection.performContextMenuAction(android.R.id.selectAll)
```

Comment:
This entire key action doesn't seem to be used anymore, so you can remove this block.